packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -316,8 +316,7 @@
 </template>
 
 <script>
-import { Buffer } from 'buffer';
```

Comment:
This `buffer` was a polyfill. I think the front-end distribution should still include the polyfill despite this code moving to `models`, but it's something to keep an eye on.

Reply:
Yeah, I had the same thought. I think we still need the polyfill either way since we're pulling in `models` as a dependency for `components`.